presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -270,7 +270,18 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
 
 .. note::
 
-    Unfortunately, there is no way to represent ``_id`` fields more fancy like ``55b151633864d6438c61a9ce``.
+    One way to present ``_id`` fields more fancy like ``55b151633864d6438c61a9ce`` is to cast them to ``VARCHAR``.
```

Comment:
How about

``` 
You can render the ``_id`` field to readable values with a cast to ``VARCHAR``:
```

And then the below sections need to be indented the same as the note paragraph .. or just remove the usage of note